share/smack/top.py

```diff
@@ -610,13 +589,15 @@ def annotate_bpl(args):
 
 
 def property_selection(args):
-    selected_props = []
-    if args.only_check_valid_deref:
-        selected_props.append('valid_deref')
-    elif args.only_check_valid_free:
-        selected_props.append('valid_free')
-    elif args.only_check_memleak:
-        selected_props.append('valid_memtrack')
+    selected_props = {}
+    if 'memory-safety' in args.check:
```

Comment:
I think that this function can simply return here.

Reply:
Done. Good point.